Dockerfile

```diff
@@ -12,16 +12,17 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY controllers/ controllers/
+COPY cmd/ cmd/
+COPY pkg/ pkg/
+
 
 # Build
 # the GOARCH has not a default value to allow the binary be built according to the host where the command
 # was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
 # the docker BUILDPLATFORM arg will be linux/arm64 when for Apple x86 it will be linux/amd64. Therefore,
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
-RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} \
+  go build -a -o manager ./cmd/manager/...
```

Comment:
Would it make more sense here to update the make file with the proper build args and just call `make build` here, and have the proper args in the make file? This would ensure that the binary built locally/ci and in the docker file match, and would reduce the places that would need to change if it's determined that the binary needs to be built differently, ie if build flags for version/commit/etc were to be added alter down the road.

Reply:
That's a great suggestion, would you be okay with filing an issue and we can handle that separately ?

Reply:
Thank you 🙇 . I am resolving this conversation